lib/companies_house/request.rb

```diff
@@ -33,6 +33,8 @@ class Request < Dry::Struct
     attribute :transaction_id, Dry.Types::String
     attribute :instrumentation, Dry.Types.Interface(:publish)
 
+    attribute :headers, Hash
```

Comment:
Should this be a `Dry.Types::Hash`? Might also want to put it above under `# Physical request attributes`.